Source/CSBackend/Rendering/OpenGL/Base/GLContextRestorer.cpp

```diff
@@ -175,7 +175,7 @@ namespace CSBackend
         }
         
         //------------------------------------------------------------------------------
-        void GLContextRestorer::OnRenderSnapshot(ChilliSource::RenderSnapshot& renderSnapshot) noexcept
+        void GLContextRestorer::OnRenderSnapshot(ChilliSource::RenderSnapshot& renderSnapshot, ChilliSource::IAllocator* frameAllocator) noexcept
```

Comment:
I moved the allocator out of the render snapshot as there is not longer a one to one mapping of ownership. However I suppose you could still time them together based on the allocator that the snapshot should use. I could revert this back easily enough